src/axom/quest/CMakeLists.txt

```diff
@@ -87,21 +87,27 @@ blt_list_append(TO quest_depends_on IF SPARSEHASH_FOUND ELEMENTS sparsehash)
 blt_list_append(TO quest_depends_on IF ENABLE_OPENMP ELEMENTS openmp)
 blt_list_append(TO quest_depends_on IF ENABLE_CUDA   ELEMENTS cuda )
 
+# Add sparsehash when available
+if(SPARSEHASH_FOUND)
```

Comment:
I would use `blt_list_append` here. Note, there was a recent fix to that macro, so, you may have to fast-forward the version of BLT

Reply:
Done. Someone already changed to using `blt_list_append` in develop.